src/as/math/Matrix4.spec.ts

```diff
@@ -423,22 +413,22 @@ describe('Maths', () => {
 				var m = testMatrices[i]
 
 				var mInverse = new Matrix4()
-				assert(mInverse.getInverse(m), 'expected inverse')
+				expect(mInverse.getInverse(m)).toBeTruthy()
 				var mSelfInverse = m.clone()
 				mSelfInverse.getInverse(mSelfInverse)
 
 				// self-inverse should the same as inverse
-				assert(matrixEquals4(mSelfInverse, mInverse), 'Passed!')
+				expect(matrixEquals4(mSelfInverse, mInverse)).toBeTruthy()
 
 				// the determinant of the inverse should be the reciprocal
-				assert(Math.abs(m.determinant() * mInverse.determinant() - 1) < 0.0001, 'Passed!')
+				expect(Math.abs(m.determinant() * mInverse.determinant() - 1)).toBeLessThanOrEqual(0.0001)
```

Comment:
While probably not ever something that would be an issue since the test is basically evaluating that the operation return close to 0, this was originally a < not a <= operation. Likewise line 430 (in the latest commit)

Reply:
Suggestion:

```ts
expect(Math.abs(m.determinant() * mInverse.determinant() - 1).toBeCloseTo(0);
```

Reply:
```ts
m.determinant() * mInverse.determinant()
```

This should be close to `1` right?

Reply:
Should be since we're subtracting 1 from it